### ISSUE TITLE
[feature request] 

### ISSUE THREAD

**Issue by aniruddha123reinforcement** [Stable-Baselines-Team/stable-baselines3-contrib#31]
**Important Note: We do not do technical support, nor consulting** and don't answer personal questions per email.
Please post your question on [reddit](https://www.reddit.com/r/reinforcementlearning/) or [stack overflow](https://stackoverflow.com/) in that case.

If you have any questions, feel free to create an issue with the tag [question].  
If you wish to suggest an enhancement or feature request, add the tag [feature request].  
If you are submitting a bug report, please fill in the following details.

If your issue is related to a custom gym environment, please check it first using:

```python
from stable_baselines3.common.env_checker import check_env

env = CustomEnv(arg1, ...)
# It will check your custom environment and output additional warnings if needed
check_env(env)
```

**Describe the bug**

**Code example**
Please try to provide a minimal example to reproduce the bug. Error messages and stack traces are also helpful.

Please use the [markdown code blocks](https://help.github.com/en/articles/creating-and-highlighting-code-blocks)
for both code and stack traces.

```python
from stable_baselines3 import ...
from sb3_contrib import ...

```

```bash
Traceback (most recent call last): File ...

```

**System Info**
Describe the characteristic of your environment:
 * Describe how the library was installed (pip, docker, source, ...)
 * Stable-Baselines3 and sb3-contrib versions
 * GPU models and configuration
 * Python version
 * PyTorch version
 * Gym version
 * Versions of any other relevant libraries

**Additional context**
[Feature request] can we expect an implementation of IQN and FQF  soon as we have a very nice implementation of QRDQN

**Comment by Miffyli** [comment]
Please fill in the template next time.

No new algorithms are planned as of right now. Purpose of this repo is for community to propose new algorithms and implement them, without need to worry about the integration with original repository.

**Comment by aniruddha123reinforcement** [comment]
Sorry about the template ...I am a newcomer to this platform ...is it
possible to pull the repository and add our own implementation?

**Comment by Miffyli** [comment]
Yes, you can clone (or fork) this repository and build upon the code. If you implement an algorithm and get the same results as in the publication the detailed the algorithm, we would be happy to include it in this repository :)
